docs/topics/modules/using_stack_reports.adoc

```diff
@@ -1,27 +1,61 @@
-+
-image::full_stack_report.png[Full Stack Report]
-+
-The `Additional components recommended by {osio}` section at the bottom has one recommendation to improve the application. It includes a link to create a new Work Item to track making the changes for this recommendation:
+* <<Stack report>>
+* <<Detail analysis of your stack components>>
+* <<Additional components recommended by {osio}>>
+
+=== Stack summary
+The Stack summary provides a synopsis of the analysis report.
+
+It displays:
+
+Security Issues:: {osio} Analytics examines the Common vulnerabilities and exposures (CVEs) of the components in the stack and highlights vulnerabilities, if any.
+
+Usage Outliers:: It analyzes your overall stack composition, compares it to other similar stacks and displays the number of components in your stack that are not commonly used in similar stacks or that rarely go together.
+
+Licenses:: It analyzes the licenses used in your stack, and displays the number of unknown licenses. It also displays the number of license conflicts in your stack.
+
+=== Detailed analysis of your stack components
+The detailed component analysis provides recommendations for relevant alternate components by analyzing all the components in the stack and their parameters such as:
+
+Versions:: It lists current, recommended, and latest available versions of the components in your stack, so that you can select a suitable version for your stack.
+
+Security Issue:: Based on the CVEs of the stack components, {osio} Analytics identifies security vulnerabilities in your stack components. The CVE scores are displayed when you move the cursor over components identified with security vulnerabilities.
+
+License:: It lists licenses for each component, flags unknown and conflicting licenses, and recommends *Alternate Components* in case of *License Outliers* (that is, licenses not used in similar stacks).
+
+The detailed analysis also helps you assess the credibility of your stack components by providing various usage statistics such as:
+
+OSIO Usage:: It tracks the number of projects in {osio} that use the component. It recommends *Alternate Components* in case of *Usage Outliers* (that is, components not used by similar stacks).
+
+GitHub Statistics:: {osio} tracks GitHub statistics, such as *Usage Count* and number of *Watchers* for the components of your stack.
+
+GitHub Dependents:: It lists all the projects dependent on this component.
+
+Tags:: It lists the tags associated with the components of your stack.
+
+You can filter the detailed component analysis report to highlight *Usage Outliers*,
+*Security Issues*,
+*Alternate Components*, *License Outliers* , *Unknown Licenses*, and *License conflicts*.
+
+=== Additional components recommended by {osio}
+{osio} recommends additional components suitable for your stack based on its analysis of similar open source stacks. These components are commonly used by similar stacks and augment your stack.
+
```

Comment:
I looked at the rendered view of this module and I'm not quite sold on these changes. IMO the detailed definitions belong in the User Guide rather than here because this is just for getting started. A basic overview of what the feature is for and how to use it should be sufficient level of details for this doc, I think. In the current rendered version, it looks like a wall of definitions and disproportionately little actual action at the end. @Preeticp @rkratky WDYT?

Reply:
I get your point and I will try to make this more compact and streamlined, have added WIP here. 
Just one thing to keep in mind. While there is no direct action steps involved in many cases, the obvious implicit action here is creating a work item to make changes in your stack based on the recommendations. I will chane some of the text to reflect this.
I thought of removing explanations for the various terms, considering that some of them are covered in the Glossary, but we do not link our E2E to the glossary at all.
I think it would be good idea to add the Glossary to the E2E, that would avoid explanations part in the E2E to a great extent, for example even in the Pipelines section. wdyt?

Reply:
With @mishaone on this. I don't think we need/want this definition list in this doc.

Instead of adding the whole  glossary to the e2e doc, I suggest we can just link to it if necessary.